integr8ly/grafana-operator
# Configurable service type

Allow setting the type of the grafana service (ClusterIP, NodePort or LoadBalancer) via an operator flag. This PR adds a new optional operator flag `--service-type` which can be used to set the type of the grafana service. Defaults to `ClusterIP` (if not set). 

The `latest` tag has been updated and contains this change.

Use case: en GCE only `NodePort` or `LoadBalancer` type services can be used with Ingresses.

Fixes #46 

Verification steps:

1. Create a copy of [operator.yaml](https://github.com/pb82/grafana-operator/blob/c6496f2a6a6d340ea6a852e3d2e34d3d739d14cc/deploy/operator.yaml) and change the service type flag to `NodePort` or `LoadBalancer`.
1. Make sure the image is pointing to the latest tag and deploy the operator.
1. Deploy Grafana
1. Check the created service and make sure its of the desired type

ping @jsanda
